### ISSUE TITLE
Start delay

### ISSUE THREAD

**Issue by shainodev** [UnamSanctam/SilentCryptoMiner#1319]
does it change anything if i remove the start delay from the miner if i inject it and the injector has a start delay already?

**Comment by UnamSanctam** [comment]
You can do that, the start delay should be as early as possible in the program to avoid the sandboxing.

**Comment by shainodev** [comment]
so it has no affect if i remove the start delay in the miner thats going to be injected? cause it is already injected in the conhost.exe so it wouldnt matter at all right, delay 15 seconds or 50 seconds since its gonna bypass avast with 50 secs ?

**Comment by UnamSanctam** [comment]
Yes that would be correct usually, though Avast could be "sandboxing" the injected one as well.

**Comment by shainodev** [comment]
> Yes that would be correct usually, though Avast could be "sandboxing" the injected one as well.

is it even possible to sandbox the injected one? since they probably take the file from the file location or what? and should i then add 50 sec delay to the miner aswell?

**Comment by UnamSanctam** [comment]
Avast technically only monitors what the program does and not actually sandboxes it for injected programs I believe.

**Comment by shainodev** [comment]
> Avast technically only monitors what the program does and not actually sandboxes it for injected programs I believe.

Ohh then i dont have to do anything i guess. also \\BaseNamedObjects\\#WATCHDOGID" what does the \\basenamedobjects\\ thing do? cause when i use it in a actual mutex it doesnt seem to work and do i have to add the basenamedobjects if my mutex in my other program doesnt have it but i think ur function needs it or what

**Comment by UnamSanctam** [comment]
It's for creating global "mutexes" when using NtCreateMutant.

**Comment by shainodev** [comment]
> It's for creating global "mutexes" when using NtCreateMutant. For normal mutexes you can just use Global.

by doing \\Global\\MUTEX ?

**Comment by UnamSanctam** [comment]
If you're using the Windows API then it would be `Global\\MUTEX` I believe.

**Comment by shainodev** [comment]
> If you're using the Windows API then it would be `Global\\MUTEX` I believe.

so in my clipper i use `CreateMutex(0, 0, Obfuscation("Global\\mutex"));` and in ur miner to check clipper mutex i use `\\BaseNamedObjects\\mutex` right?

**Comment by UnamSanctam** [comment]
Yes that should be correct I believe.

**Comment by shainodev** [comment]
thank you
